src/components/InputBlock/InputBlock.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components/macro'
+
+const InputBlock = ({ set, necessary, maxScore }) => {
+  return (
+    <InputWrapper>
+      <span>Set {set} </span>
+      <label htmlFor={`user-result-set-${set}`}>User result</label>
+      <input
+        type="number"
+        id={`user-result-set-${set}`}
+        min="0"
+        max={maxScore}
+        name={`player1Set${set}`}
+        required={necessary}
+      />{' '}
+      : <label htmlFor={`oppenent-result-set-${set}`}>Opponent result</label>
+      <input
+        type="number"
+        id={`oppenent-result-set-${set}`}
+        min="0"
+        max={maxScore}
+        name={`player2Set${set}`}
+        required={necessary}
+      />
+    </InputWrapper>
```

Comment:
I would keep the naming as is especially with bools. 
```suggestion
const InputBlock = ({ set, required, maxScore }) => {
  return (
    <InputWrapper>
      <span>Set {set} </span>
      <label htmlFor={`user-result-set-${set}`}>User result</label>
      <input
        type="number"
        id={`user-result-set-${set}`}
        min="0"
        max={maxScore}
        name={`player1Set${set}`}
        required={required}
      />{' '}
      : <label htmlFor={`oppenent-result-set-${set}`}>Opponent result</label>
      <input
        type="number"
        id={`oppenent-result-set-${set}`}
        min="0"
        max={maxScore}
        name={`player2Set${set}`}
        required={required}
      />
    </InputWrapper>
```

Reply:
I reverted to the original solution, thank you for the feedback 😇

Reply:
My pleasure 😸